### DISCUSSION TITLE
Limiting KTL to one view

### DISCUSSION THREAD

**Discussion by Kaitydid789** [cortexrd/Knack-Toolkit-Library#266] [Q&A] [answered]
Hey there! My first time posting anything ever on Github (or really using it at all), so apologies in advance if I say anything dumb. :)

I raised this in the Knack Community Forum today. I'm interested in using the bulk edit feature of KTL just for one table in the app I'm creating. I'm trying to create an attendance tracker for an after-school program. See the first screenshot below. Bulk edit gives the coach an easy way to edit all kids' records in the table for a given day.

But I don't want bulk edit to be enabled for other tables, like with the simple table in the 2nd screenshot.

Thanks in advance for your help!

<img width="1330" alt="Screen Shot 2024-03-10 at 7 30 13 PM" src="https://github.com/cortexrd/Knack-Toolkit-Library/assets/162930343/bf998820-36e9-4681-9f5e-beaa8eef844e">
<img width="468" alt="Screen Shot 2024-03-10 at 7 31 55 PM" src="https://github.com/cortexrd/Knack-Toolkit-Library/assets/162930343/1ea7f95c-e079-4df2-9689-56e177c43327">

**Comment by cortexrd** [comment]
HI @Kaitydid789,

If you only want to enable Bulk Edit (or any other bulk op), you can use the **_ebo** keyword in that grid.
See documentation here:  [_ebo](https://github.com/cortexrd/Knack-Toolkit-Library/wiki/Keywords#_eboedit-copy-delete)

In your case, this is the correct syntax:

`_ebo=edit`

to enable Bulk Edit only.

You can also enable it for specific roles only.  Or any roles except Role1, Role2, etc.

The KTL is extremely flexible, as you can see.

Enjoy,
Norm

**Reply by Kaitydid789** [reply, marked as answer]
Ahhhh got it - that worked! Thank you!
